packages/core/src/Cardano/util/computeImplicitCoin.ts

```diff
@@ -30,6 +30,14 @@ const stakeCredentialInRewardAccounts = (
   return rewardAccounts.includes(Address.RewardAccount.fromCredential(stakeCredential, networkId));
 };
 
+const stakeAddressInRewardAccounts = (
+  stakeAddress: Address.RewardAccount,
+  rewardAccounts: Address.RewardAccount[]
+): boolean => {
+  if (rewardAccounts.length === 0) return true;
```

Comment:
Sorry for nitpicking. I find this line confusing. What I expect from this function is: if there are no reward accounts, then stake address is not included. But this line makes it: if there are no reward accounts, then stake address is included.
My suggestion would be to remove this line, as the empty reward accounts scenario is already captured at line 199:
`const withdrawalsTotal = hasRewardAccount ? getOwnWithdrawals(withdrawals, rewardAccounts) : withdrawalsAll;`

Reply:
Completely valid comment, not a nit. Thank you! It was a mistake. This utility became useless in the end, so I inlined it.